### DISCUSSION TITLE
Discussions section on collaboration guide

### DISCUSSION THREAD

**Discussion by Jorge-Rodriguez** [ansible-collections/ansible.mysql#124] [Show and tell]
I've added the collaboration guide to our [wiki](https://github.com/ansible-collections/community.mysql/wiki/Contributing), mostly copied word for word from [the PR](https://github.com/ansible-collections/community.mysql/pull/112) by  @Andersson007 
I have also added a [section about the repo's discussion's space](https://github.com/ansible-collections/community.mysql/wiki/Contributing#discussions).

It would be great to hear your comments on it.

**Comment by Andersson007** [comment]
@Jorge-Rodriguez LGTM, thanks! You could add the section to the CONTRIBUTING.md itself.
